docs/src/test-retries-js.md

```diff
@@ -134,6 +134,42 @@ test('my test', async ({ page }, testInfo) => {
 });
 ```
 
+You can specify retries for a specific group of tests or a single file with [`method: Test.describe.configure`].
+
+```js tab=js-js
+const { test, expect } = require('@playwright/test');
+
+test.describe(() => {
+  // All tests in this describe group will get 2 retry attempts.
```

Comment:
One of the snippets should contain top-level tests instead of test.describe?

Reply:
These are js and ts snippets. I decided to show the describe version, because it's more useful based on the filed issues.